index.js

```diff
@@ -110,7 +110,7 @@ function parseOpts (opts) {
   if (!opts.ignore) opts.ignore = []
   opts.ignore = opts.ignore.concat(DEFAULT_IGNORE)
 
-  setGlobals(opts.globals || opts.global)
+  setGlobals(opts.global)
```

Comment:
This change is not right. I want to support either option name when people `require('standard')` in node.js. Even if the command line uses only `global`, we should keep the other.